cdap-docs/admin-manual/source/security/authorization.rst

```diff
@@ -304,90 +312,158 @@ Streams
    * - Operation
      - Privileges Required
    * - Create
-     - *WRITE* (on the namespace)
+     - *ADMIN*
    * - Retrieving events
-     - *READ* (on the stream and the namespace)
-   * - Retrieving properties
-     - At least one of *READ*, *WRITE*, *ADMIN*, or *EXECUTE*
+     - *READ*
    * - Sending events to a stream (sync, async, or batch)
-     - *WRITE* (on the stream) and *READ* (on the namespace)
+     - *WRITE*
    * - Drop
      - *ADMIN*
    * - Drop-all in the namespace
-     - *ADMIN* (on the namespace)
+     - *ADMIN* on all streams in the namespace
    * - Update
      - *ADMIN*
    * - Truncate
      - *ADMIN*
-   * - List
-     - Only returns those artifacts on which user has at least one of *READ, WRITE, EXECUTE,* or *ADMIN*
-   * - View
+   * - List/View
+     - Only returns those streams on which user has at least one of *READ, WRITE, EXECUTE,* or *ADMIN*
+   * - Get
      - At least one of *READ, WRITE, EXECUTE,* or *ADMIN*
 
+.. _security-authorization-policies-principal:
 
-.. _security-bootstrapping-authorization:
+Kerberos Principal
+------------------
 
-Bootstrapping Authorization
-===========================
-When CDAP is first started with authorization enabled, no users are granted privileges on
-any CDAP entities. Without any privileges, it can be impossible to bootstrap CDAP (create
-a new namespace or create entities in the *default* namespace) unless an
-external interface (such as `Hue <http://gethue.com/>`__) is used for a supported
-authorization extension (such as :ref:`Integrations: Apache Sentry <apache-sentry>`).
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Operation
+     - Privileges Required
+   * - Deploy an app to impersonate a kerberos principal
+     - *ADMIN* on the principal
+   * - Create a namespace with owner prinicpal
+     - *ADMIN* on the principal
+   * - Create a dataset with owner prinicpal
+     - *ADMIN* on the principal
+   * - Create a stream with owner prinicpal
+     - *ADMIN* on the principal
 
-To make this bootstrap process easier, during startup the CDAP Master issues these grants to
-select users:
 
-- The user that runs the CDAP Master is granted *ADMIN* on the CDAP instance, so that the
-  *default* namespace can be created by that user if it does not already exist.
+.. _security-pre-grant-wildcard-privilege:
+
+Pre-grant and Wildcard Privileges
+=================================
+The new authorization model requires pre-granted *ADMIN* privilege on all entity operations.
```

Comment:
This line seema a bit misleading ADMIN is not needed for all operation just new entity creation. Can you rewrite it.

Reply:
👍